javascript/code-challenges/challenges-06.test.js

```diff
@@ -0,0 +1,180 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named getNames that, given an array of people objects, uses map to return an array of names reversed.
+
+For example:
+[
+{
+  name: 'lloyd',
+  age: 32,
+  shoeSize: 12
+},
+{
+  name: 'jamie',
+  age: 21,
+  shoeSize: 8
+}
+]
+
+Returns: ['dyoll', 'eimaj'];
+------------------------------------------------------------------------------------------------ */
+
+const getNames = (arr) => {
+
+ return arr.map(item => item.name.split('').reverse().join(''));
+  
```

Comment:
Just for kicks and giggles you could do an implicit return to get the entire solution on one line. During my cohort there were two students who were always trying to do as many challenges as possible in a single line.

```js
const getNames = arr => arr.map(item => item.name.split('').reverse().join(''));
```